cit3.0-web/src/components/ChangeView/ChangeView.js

```diff
@@ -1,7 +1,7 @@
 import { useMap } from "react-leaflet";
 
 export default function ChangeView({ center, zoom }) {
-  if (center && zoom) {
+  if (center[0] !== 54.1722) {
```

Comment:
Just for my own understanding, what does this fix?

Reply:
This prevents zooming in on the initial render.